tools/Dockerfile

```diff
@@ -1,11 +1,10 @@
-
 FROM        ocaml/opam2:debian-9-ocaml-4.07
 MAINTAINER  Christian Lindig <christian.lindig@citrix.com>
 
-COPY . /tmp/xs-opam
+COPY --chown=opam:opam . /tmp/xs-opam
 
 RUN sudo apt-get update
 RUN opam repo remove --all default \
  && opam repo add xs-opam file:///tmp/xs-opam \
  && opam depext -y xs-toolstack \
- && opam install -j 4 xs-toolstack
+ && opam install xs-toolstack
```

Comment:
Did you intend to remove the `-j` here?

Reply:
Actually, I changed it but didn't intend to commit it. In my machine it uses `11` as default, from the docs:

```
       -j JOBS, --jobs=JOBS
           Set the maximal number of concurrent jobs to use. The default value
           is calculated from the number of cores. You can also set it using
           the $OPAMJOBS environment variable.
```
[default](https://github.com/ocaml/opam/blob/9778ada9360b9e32ff0ae025999d6f77831dc211/src/state/opamStateConfig.ml#L35)

Reply:
Ok then removing it is probably better since it allows more parallelism